third_party/blink/renderer/core/css/style_recalc_change.cc

```diff
@@ -28,19 +29,42 @@ bool StyleRecalcChange::TraverseChild(const Node& node) const {
 }
 
 bool StyleRecalcChange::ShouldRecalcStyleFor(const Node& node) const {
-  if (flags_ & kSuppressRecalc)
+  recordreplay::Assert("[RUN-1219-1706] StyleRecalcChange::ShouldRecalcStyleFor #0 %d",
+    node.RecordReplayId());
+  if (flags_ & kSuppressRecalc) {
+    recordreplay::Assert("StyleRecalcChange::ShouldRecalcStyleFor %d suppressRecalc",
```

Comment:
Instead of adding all these individual `Assert`s, you could just add all individual branch condition inputs into the top `Assert`